### PR TITLE
Add config generation for simple user namespace testing.

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: nsinit/config.go [hunk 1]
```diff
@@ -70,6 +71,22 @@ func modify(config *configs.Config, context *cli.Context) {
 	config.AppArmorProfile = context.String("apparmor-profile")
 	config.ProcessLabel = context.String("process-label")
 	config.MountLabel = context.String("mount-label")
+
+	userns_uid := context.Int("userns-root-uid")
+	maxUid := 1 << 32
```
Comment (vmarmol) [inline nsinit/config.go]: Is this meant to be an int32? This is undefined for them (shifting word size).

Comment (mrunalp) [inline nsinit/config.go]: Yeah, want to set it to 2^32. Seems to be working but if it is undefined then need to change this. Thanks.

Comment (vmarmol) [inline nsinit/config.go]: if we cast the 1 as an int64 it should be fine :)

Comment (mrunalp) [inline nsinit/config.go]: Are you suggesting?
```
maxUid := int(int64(1) << 32)
```

Comment (vmarmol) [inline nsinit/config.go]: Ah, well 2^32 is not a valid integer (int32). The largest unsigned integer is 2^32 -1. Should this be max unsigned int or max signed int? (both int32)

Comment (mrunalp) [inline nsinit/config.go]: It is defined as unsigned uint32 in linux but is represented as int in golang.

**Comment by mrunalp** [comment]
Updated to use math.MaxInt32.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**